docs/releases/1.11.1.txt

```diff
@@ -9,4 +9,6 @@ Django 1.11.1 fixes several bugs in 1.11.
 Bugfixes
 ========
 
-* ...
+* Made migrations respect ``Index``\'s ``name`` argument. If you created a
```

Comment:
Ah, sorry. I thought that was some fancy c&p magic that happened when posting to Github.